### PR TITLE
models, started handlebars, empty public files, one user-route

### PR DESCRIPTION
**Pull request by jpmare29**
I did one of the user routes like how the module has them set up, if you have questions on what's going on with it Brittany just lemme know.

### CODE REVIEW

File: controllers/api/movie-routes.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+const { Movie } = require('../../models');
 
 router.get("/movies", (req, res) => {
-    let results = books;
-    if (req.query) {
-      results = filterByQuery(req.query, results);
-    }
-    res.json(results);
+    Movie.findAll()
+    .then(movieData => {
```
Comment (bgahan) [inline controllers/api/movie-routes.js:6]: Is `movieData` supposed to be what the frontend will pass in to search off of? Like title/genre/etc.?

File: controllers/api/movie-routes.js [hunk 2]
```diff
@@ -1,11 +1,15 @@
...
+      res.json(movieData);
+    })
+    .catch(err => {
+      console.log(err);
+      res.status(500).json(err);
+    })
   });
 
 router.post("/movies", (req, res) => {
```
Comment (bgahan) [inline controllers/api/movie-routes.js:15]: Do we need to add something similar to this route to add the data to the database?

**Review by bgahan — APPROVED** [review]
Looks good, just had a couple of questions that I was curious about.

File: controllers/api/movie-routes.js (hunk 1 above)
Comment (jpmare29) [reply controllers/api/movie-routes.js:6]: So findAll is a promise based function, when you append that .then() it’s gonna receive an object returned by findAll as a parameter you can name that parameter anything in the callback function just as long as you maintain the same name

File: controllers/api/movie-routes.js (hunk 2 above)
Comment (jpmare29) [reply controllers/api/movie-routes.js:15]: Are you asking about the catch? Cause that’s just an error handler, not sure if you meant something else

**Pull request merged**